packages/plugin-commands-publishing/test/publish.ts

```diff
@@ -351,12 +351,16 @@ test('convert specs with workspace protocols to regular version ranges', async (
         'is-positive': '1.0.0',
         'lodash.delay': '~4.1.0',
         odd: 'workspace:is-odd@*',
+        rd: 'workspace:ramda@^',
```

Comment:
I'm having problem with this dependency, if I remove `rd: 'workspace:ramda@^'` the test passes. The error message says that `is-negative` could not be found.

```
Cannot resolve workspace protocol of dependency "is-negative" because this dependency is not installed
```

Do you know how could I troubleshoot this error?

Reply:
I think it's fixed now. Among the things that I modified to make it work, I had to change package versions to those specified in `verdaccio/node_modules/storage-cache`.

Is this ok? Or am I just fooling myself?